src/test/librbd/test_mock_fixture.h

```diff
@@ -25,7 +25,11 @@ ACTION_P(CopyInBufferlist, str) {
 }
 
 ACTION_P2(CompleteContext, r, wq) {
+#if defined(__clang__)
+  ContextWQ *context_wq = static_cast<ContextWQ *>(wq);
```

Comment:
@tchaikov @dillaman 
Any suggestions on how to move this forward?

1) Rip open all the other calls and start using nullptr everywhere?
2) Let this part be ifdef-ed 
2a) Let this part be ifdef-ed for the time being, and work on 1)